components/alert/alert.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ditsy UI</title>
+    <link
+      rel="shortcut icon"
+      href="../../assets/favicon.ico"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="../../index.css" />
+  </head>
+  <body>
+    <navbar class="topnav">
+      <a class="link" href="/"
+        ><img class="header-image" src="../../assets/logo.png"
+      /></a>
+      <div class="topnav-options">
+        <span>Docs</span><span>Examples</span><span>Releases</span>
+      </div>
+    </navbar>
+
+    <div class="sidebar-and-main-container">
+      <div class="sidebar">
+        <h3 class="sidebar-heading">GETTING STARTED</h3>
+        <ul>
+          <a href="../introduction/introduction.html"
+            ><li class="list-item">Introduction</li></a
+          >
+          <a href="../colors/colors.html"><li class="list-item">Colors</li></a>
+          <a href="../typography/typography.html"
+            ><li class="list-item">Typography</li></a
+          >
+          <a href="../open_source/open_source.html"
+            ><li class="list-item">Open Source</li></a
+          >
+        </ul>
+        <h3 class="sidebar-heading">COMPONENTS</h3>
+        <ul>
+          <a href=""><li class="list-item">Avatar</li></a>
+          <a href="./alert.html"><li class="list-item">Alert</li></a>
+          <a href="/"><li class="list-item">Badge</li></a>
+          <a href="/"><li class="list-item">Button</li></a>
+          <a href="/"><li class="list-item">Card</li></a>
+          <a href="/"><li class="list-item">Image</li></a>
+          <a href="/"><li class="list-item">Input</li></a>
+          <a href="/"><li class="list-item">List</li></a>
+          <a href="/"><li class="list-item">Toast</li></a>
+          <a href="/"><li class="list-item">Rating</li></a>
+        </ul>
+      </div>
+
+      <main class="main-container">
+        <h1 class="main-heading">Alert</h1>
+        <div class="divider"></div>
+
+        <section>
+          <h2>Default Alert</h2>
+          <div class="alert success">Success</div>
```

Comment:
Don't wrap text inside the div element instead use a `p` tag. This is semantically not correct.